recipes/server.rb

```diff
@@ -66,3 +66,13 @@
 sysctl_param 'vm.swappiness' do
   value 0
 end
+
+cookbook_file '/usr/local/bin/mysql-accounting' do
+  source 'mysql-accounting'
+  mode '0755'
+end
```

Comment:
I'm pondering if its more useful to just directly put this script into the crontab instead of copying a file over. We could do it with ``command %w{ }`` as shown in the [upstream docs](https://docs.chef.io/resource_cron.html) and embed it into the recipe. It's small enough that it might be better that way? I doubt we need to have this in a regular path.